### ISSUE TITLE
Indexing/ Identifying movies & some issues

### ISSUE THREAD

**Issue by Gougoule** [MizzleDK/Mizuu#504]
Hello, first of all, i just discovered your app and i LOVE it. simply the best appz out there and i tested quite a lot. Perfect for my Ouya that what dying :)

1. just a small (but very helpful) improvement in the indexation rules that you could make ! I understand that when mizuu scan a remote directory (in samba for my case) it index all the files that it can recognize...this work great for a lot of cases but not always....let me take an example :)

Ghost.In.The.Shell.2.0.720p.BluRay.x264.mkv => this is recognized :)
Ghost.In.The.Shell.2.0.720p.BluRay.x264-MANGACiTY.mkv => this is not because it considers that Mangacity is part of the title of the movie (which if of course wrong). i have others example but i think you got the idea

2. This would not be a big issue if the file where in the not identified list (in the following case, this list is empty). instead of this Mizuu takes all the files that it cannot recognized and put them under one "random" (actually something that looks like the name of the directory where the files are stored) title...then I have to go to this title movie (that is not part of my library) to identify each movie manually....all the manual work is of course deleted when I rescan my library :(

In the bellow screen I do not have the Mickey McGuire movie on my library. 

When I try to identify the movie, here is what I have (second screen)

either I'm doing something bad, either this is a bug :)

![img_20150102_143309](https://cloud.githubusercontent.com/assets/10370660/5596396/42ea3d9e-928d-11e4-8f44-1ab7d8e44620.jpg)

![img_20150102_143257](https://cloud.githubusercontent.com/assets/10370660/5596381/f63861a6-928c-11e4-8287-71f549f2aeb8.jpg)

thanks for your feedback / fix !

KR,

**Comment by MizzleDK** [comment]
Hi!

It's impossible to maintain a list of release groups and filter it away. If the filename had included a release year following the title, it would have worked just fine.

The many movies linked to "Mickey's Movies" is due to your folder structure. Mizuu will - by default - attempt to identify movies based on the filename, and if that fails, it'll attempt using the parent folder name. Yours is called "-= Movies =-", and that's what's causing it to match it as "Mickey's Movies".

I see that there are problems with all the filenames in your first image. Once you fix these, it'll work just fine. Please see this regarding naming of movie files:
https://github.com/MizzleDK/Mizuu/wiki/Naming-conventions-for-movies

In conclusion, it's not a bug - just incorrectly named files :-)

**Comment by Gougoule** [comment]
thanks for your quick feedback. now it's now clearer. For me the group name is important to keep. is there any way to update the naming rule to discard everything after the title of the movie ?

**Comment by MizzleDK** [comment]
Yes, you can include a release year or the IMDb ID.

For example:
Ghost.In.The.Shell.2.0 (2008) 720p.BluRay.x264-MANGACiTY.mkv
Ghost.In.The.Shell.2.0.720p.BluRay.x264-MANGACiTY (tt1260502).mkv

Both would work just fine.

**Comment by Gougoule** [comment]
that's a quick answer :) and again crystal clear. thanks for your feedback. I will either rename or include the IMDB It :) 

KR and please keep up the excellent work with your app :)

**Comment by MizzleDK** [comment]
Thank you! :-)

Closing this.
